pkg/trace/obfuscate/cache.go

```diff
@@ -56,14 +56,14 @@ func newMeasuredCache() *measuredCache {
 	cfg := &ristretto.Config{
 		// We know that the maximum allowed resource length is 5K. This means that
 		// in 5MB we can store a minimum of 1000 queries.
-		MaxCost: 5_000_000,
```

Comment:
☹️ Why? This is much nicer and more readable.

Reply:
You were right, this is an issue with my gofmt not being up to date with go 1.13. removing